File: Ejercicios en clases/poo java/Two java/library/Circle.java
   public class Circle extends GeometricFigure {
    private double radius;

    public Circle(double radius) {
        super("Circle");
        this.radius = radius;
    }

    public double getRadius() {
        return radius;
    }

    @Override
    public double calculateArea() {
        return Math.PI * radius * radius;
    }
}

File: Ejercicios en clases/poo php/Two php/Library/circle.php
<?php

class Circle extends geometricFigures {
    private float $radius;
    private float $calculate;

    public function __construct(float $radius) {
        parent::__construct("Circle");
        $this->radius = $radius;
    }

    public function getRadius(): float {
        return $this->radius;
    }

    public function calculateArea(): float {
        $this->calculate = pi() * pow($this->radius,2);
        return $this->calculate;
    }
}

